libtransact/Cargo.toml

```diff
@@ -53,6 +53,7 @@ serde_yaml = { version = "0.8", optional = true }
 sha2 = "0.8"
 tar = { version = "0.4", optional = true }
 uuid = { version = "0.7", optional = true, features = ["v4"] }
+yaml-rust =  { version = "0.4", optional = true }
```

Comment:
Is there a reason for using `yaml-rust` over `serde_yaml`?

Reply:
its what was by the original smallbank_workload code.